A test sometimes break the Travis build

Sometimes one of the Notifications tests breaks the Travis CI build:
```
=================================== FAILURES ===================================
____________________ TestNotifications.test_2_notifications ____________________
self = <test_notifications.TestNotifications testMethod=test_2_notifications>
    def test_2_notifications(self):
        """ Test notifications sent in normal mode
    
            :return: None
            """
        self.print_header()
        self.setup_with_file('cfg/cfg_default.cfg')
    
        host = self.schedulers['scheduler-master'].sched.hosts.find_by_name("test_host_0")
        host.checks_in_progress = []
        host.act_depend_of = []  # ignore the router
        host.event_handler_enabled = False
    
        svc = self.schedulers['scheduler-master'].sched.services.find_srv_by_name_and_hostname(
            "test_host_0", "test_ok_0")
        # To make tests quicker we make notifications send very quickly
        svc.notification_interval = 0.001
        svc.checks_in_progress = []
        svc.act_depend_of = []  # no hostchecks on critical checkresults
        svc.event_handler_enabled = False
    
        self.scheduler_loop(1, [[host, 0, 'UP'], [svc, 0, 'OK']])
        time.sleep(0.1)
        assert 0 == svc.current_notification_number, 'All OK no notifications'
        self.assert_actions_count(0)
    
        self.scheduler_loop(1, [[svc, 2, 'CRITICAL']])
        time.sleep(0.1)
        assert "SOFT" == svc.state_type
        assert 0 == svc.current_notification_number, 'Critical SOFT, no notifications'
        self.assert_actions_count(0)
    
        self.scheduler_loop(1, [[svc, 2, 'CRITICAL']])
        time.sleep(0.1)
        assert "HARD" == svc.state_type
        assert 1 == svc.current_notification_number, 'Critical HARD, must have 1 ' \
                                                             'notification'
        self.assert_actions_count(2)
    
        self.scheduler_loop(1, [[svc, 2, 'CRITICAL']])
        time.sleep(0.1)
        assert svc.current_notification_number == 2
        self.assert_actions_count(3)
    
        self.scheduler_loop(1, [[svc, 2, 'CRITICAL']])
        time.sleep(0.1)
        assert svc.current_notification_number == 3
        self.assert_actions_count(4)
    
        now = time.time()
        cmd = "[%lu] DISABLE_CONTACT_SVC_NOTIFICATIONS;test_contact" % now
        self.schedulers['scheduler-master'].sched.run_external_command(cmd)
        self.scheduler_loop(1, [[svc, 2, 'CRITICAL']])
        time.sleep(0.1)
        assert svc.current_notification_number == 3
        self.assert_actions_count(4)
    
        now = time.time()
        cmd = "[%lu] ENABLE_CONTACT_SVC_NOTIFICATIONS;test_contact" % now
        self.schedulers['scheduler-master'].sched.run_external_command(cmd)
        self.scheduler_loop(1, [[svc, 2, 'CRITICAL']])
        assert svc.current_notification_number == 4
        self.assert_actions_count(5)
    
        self.scheduler_loop(1, [[svc, 0, 'OK']])
        time.sleep(0.1)
        assert 0 == svc.current_notification_number
>       self.assert_actions_count(5)
test_notifications.py:208: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
alignak_test.py:532: in assert_actions_count
    for idx, b in enumerate(actions))))
E   AssertionError: 5 != 6 : Not found expected number of actions:
E   actions_logs=[[[
E   	0 = creation: 1479868123.55, is_a: notification, type: PROBLEM, status: scheduled, planned: 1479868123, command: /notifier.pl --hostname test_host_0 --servicedesc test_ok_0 --notificationtype PROBLEM --servicestate CRITICAL --serviceoutput CRITICAL --longdatetime Wed 23 Nov 02:28:43 UTC 2016 --serviceattempt 2 --servicestatetype HARD
E   	1 = creation: 1479868123.66, is_a: notification, type: PROBLEM, status: scheduled, planned: 1479868123.61, command: /notifier.pl --hostname test_host_0 --servicedesc test_ok_0 --notificationtype PROBLEM --servicestate CRITICAL --serviceoutput CRITICAL --longdatetime Wed 23 Nov 02:28:43 UTC 2016 --serviceattempt 2 --servicestatetype HARD
E   	2 = creation: 1479868123.77, is_a: notification, type: PROBLEM, status: scheduled, planned: 1479868123.67, command: /notifier.pl --hostname test_host_0 --servicedesc test_ok_0 --notificationtype PROBLEM --servicestate CRITICAL --serviceoutput CRITICAL --longdatetime Wed 23 Nov 02:28:43 UTC 2016 --serviceattempt 2 --servicestatetype HARD
E   	3 = creation: 1479868124.0, is_a: notification, type: PROBLEM, status: scheduled, planned: 1479868123.96, command: /notifier.pl --hostname test_host_0 --servicedesc test_ok_0 --notificationtype PROBLEM --servicestate CRITICAL --serviceoutput CRITICAL --longdatetime Wed 23 Nov 02:28:43 UTC 2016 --serviceattempt 2 --servicestatetype HARD
E   	4 = creation: 1479868124.02, is_a: notification, type: PROBLEM, status: scheduled, planned: 1479868124.02, command: /notifier.pl --hostname test_host_0 --servicedesc test_ok_0 --notificationtype PROBLEM --servicestate CRITICAL --serviceoutput CRITICAL --longdatetime Wed 23 Nov 02:28:44 UTC 2016 --serviceattempt 2 --servicestatetype HARD
E   	5 = creation: 1479868124.03, is_a: notification, type: RECOVERY, status: scheduled, planned: 1479868124, command: /notifier.pl --hostname test_host_0 --servicedesc test_ok_0 --notificationtype RECOVERY --servicestate OK --serviceoutput OK --longdatetime Wed 23 Nov 02:28:44 UTC 2016 --serviceattempt 1 --servicestatetype HARD
E   ]]]
----------------------------- Captured stdout call -----------------------------
```

--- a/test/test_notifications.py
+++ b/test/test_notifications.py
@@ -154,58 +154,59 @@ class TestNotifications(AlignakTest):
         svc = self.schedulers['scheduler-master'].sched.services.find_srv_by_name_and_hostname(
             "test_host_0", "test_ok_0")
         # To make tests quicker we make notifications send very quickly
-        svc.notification_interval = 0.001
+        svc.notification_interval = 0.01 # so it's 0.6 second
         svc.checks_in_progress = []
         svc.act_depend_of = []  # no hostchecks on critical checkresults
         svc.event_handler_enabled = False
 
         self.scheduler_loop(1, [[host, 0, 'UP'], [svc, 0, 'OK']])
-        time.sleep(0.1)
-        assert 0 == svc.current_notification_number, 'All OK no notifications'
+        time.sleep(0.7)
+        assert svc.current_notification_number == 0, 'All OK no notifications'
         self.assert_actions_count(0)
 
         self.scheduler_loop(1, [[svc, 2, 'CRITICAL']])
-        time.sleep(0.1)
+        time.sleep(0.7)
         assert "SOFT" == svc.state_type
-        assert 0 == svc.current_notification_number, 'Critical SOFT, no notifications'
+        assert svc.current_notification_number == 0, 'Critical SOFT, no notifications'
         self.assert_actions_count(0)
 
         self.scheduler_loop(1, [[svc, 2, 'CRITICAL']])
-        time.sleep(0.1)
+        time.sleep(0.7)
         assert "HARD" == svc.state_type
-        assert 1 == svc.current_notification_number, 'Critical HARD, must have 1 ' \
-                                                             'notification'
         self.assert_actions_count(2)
+        assert svc.current_notification_number == 1, 'Critical HARD, must have 1 ' \
+                                                     'notification'
 
         self.scheduler_loop(1, [[svc, 2, 'CRITICAL']])
-        time.sleep(0.1)
-        assert svc.current_notification_number == 2
+        time.sleep(0.7)
         self.assert_actions_count(3)
+        assert svc.current_notification_number == 2
 
         self.scheduler_loop(1, [[svc, 2, 'CRITICAL']])
-        time.sleep(0.1)
-        assert svc.current_notification_number == 3
+        time.sleep(0.7)
         self.assert_actions_count(4)
+        assert svc.current_notification_number == 3
 
         now = time.time()
         cmd = "[%lu] DISABLE_CONTACT_SVC_NOTIFICATIONS;test_contact" % now
         self.schedulers['scheduler-master'].sched.run_external_command(cmd)
         self.scheduler_loop(1, [[svc, 2, 'CRITICAL']])
-        time.sleep(0.1)
-        assert svc.current_notification_number == 3
+        time.sleep(0.7)
         self.assert_actions_count(4)
+        assert svc.current_notification_number == 3
 
         now = time.time()
         cmd = "[%lu] ENABLE_CONTACT_SVC_NOTIFICATIONS;test_contact" % now
         self.schedulers['scheduler-master'].sched.run_external_command(cmd)
         self.scheduler_loop(1, [[svc, 2, 'CRITICAL']])
-        assert svc.current_notification_number == 4
+        time.sleep(0.7)
         self.assert_actions_count(5)
+        assert svc.current_notification_number == 4
 
         self.scheduler_loop(1, [[svc, 0, 'OK']])
-        time.sleep(0.1)
+        time.sleep(0.7)
         assert 0 == svc.current_notification_number
-        self.assert_actions_count(5)
+        self.assert_actions_count(6)
 
     def test_3_notifications(self):
         """ Test notifications of service states OK -> WARNING -> CRITICAL -> OK

